xbmc/utils/ProgressJob.h

```diff
@@ -158,6 +158,6 @@ class CProgressJob : public CJob
   bool m_autoClose = true;
   bool m_updateProgress = true;
   bool m_updateInformation = true;
-  mutable CGUIDialogProgressBarHandle* m_progress;
-  mutable CGUIDialogProgress* m_progressDialog;
+  CGUIDialogProgressBarHandle* m_progress = nullptr;
+  CGUIDialogProgress* m_progressDialog = nullptr;
```

Comment:
Also, our style guide changed since the above lines were written, and we decided it's better follow it for new code over code consistency, so can you do:

```c++
  CGUIDialogProgressBarHandle* m_progress{nullptr};
```

Sorry for such a nit. Thanks!